src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -317,7 +316,9 @@ def redis(self) -> "ExternalConnectionCommunicator":
     @property
     def _device_info_dict(self) -> Dict:
         """Return the asset info."""
-        return {}
+        return {
+            **self._settlement_market_strategy.get_unsettled_deviation_dict(self)
```

Comment:
What is expected to happen when a mixed in class doesn't define this `_settlement_market_strategy` ?

Reply:
Exception. However, as discussed offline, this is how mixins work, and that's why we use them only in very limited context (essentially only for classes that inherit from BaseStrategy / BidEnabledStrategy, therefore we know that this member will be part of the concrete class).